x-pack/plugins/canvas/canvas_plugin_src/functions/common/image.test.js

```diff
@@ -7,61 +7,67 @@
 
 import expect from '@kbn/expect';
 import {
-  elasticLogo,
-  elasticOutline,
+  getElasticLogo,
+  getElasticOutline,
+  functionWrapper,
 } from '../../../../../../src/plugins/presentation_util/common/lib';
-// import { image } from './image';
+import { image } from './image';
 
 // TODO: the test was not running and is not up to date
-describe.skip('image', () => {
-  const fn = jest.fn();
+describe('image', () => {
+  const fn = functionWrapper(image);
 
-  it('returns an image object using a dataUrl', () => {
-    const result = fn(null, { dataurl: elasticOutline, mode: 'cover' });
+  let elasticLogo;
+  let elasticOutline;
+  beforeEach(async () => {
+    elasticLogo = (await getElasticLogo()).elasticLogo;
+    elasticOutline = (await getElasticOutline()).elasticOutline;
+  });
+
+  it('returns an image object using a dataUrl', async () => {
+    const result = await fn(null, { dataurl: elasticOutline, mode: 'cover' });
     expect(result).to.have.property('type', 'image');
   });
 
   describe('args', () => {
     describe('dataurl', () => {
-      it('sets the source of the image using dataurl', () => {
-        const result = fn(null, { dataurl: elasticOutline });
+      it('sets the source of the image using dataurl', async () => {
+        const result = await fn(null, { dataurl: elasticOutline });
         expect(result).to.have.property('dataurl', elasticOutline);
       });
 
-      it.skip('sets the source of the image using url', () => {
+      it.skip('sets the source of the image using url', async () => {
```

Comment:
Can you check to see if this test can be unskipped?

Reply:
It can't be, sorry. I wished to remove `skip` in my `Expression image` PR, but for now, it is not possible.